Where_Is_My_Wife/Assets/_Main/Scripts/UI/Settings/SettingSelection.cs

```diff
@@ -14,13 +14,13 @@ protected virtual void Start()
             _uiInputEvent = InputEventManager.Instance.UIInputEvent;
         }
     
-        public virtual void OnSelect(BaseEventData eventData)
+        public override void OnSelect(BaseEventData eventData)
         {
             base.OnSelect(eventData);
             SubscribeToActions();
         }
     
-        public virtual void OnDeselect(BaseEventData eventData)
+        public override void OnDeselect(BaseEventData eventData)
         {
             base.OnDeselect(eventData);
             SelectedUnsubscribeFromActions();
```

Comment:
Good catch. Since we are inheriting the interfaces implementation from MenuButton, we can delete them from this class.